charts/spire/charts/spire-server/values.yaml

```diff
@@ -225,7 +225,7 @@ federation:
 
 ca_subject:
   ## @param ca_subject.country Country for Spire server CA
-  country: NL
+  country: ARPA
```

Comment:
https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes

Which country does ARPA represent?

Reply:
https://www.digicert.com/kb/ssl-certificate-country-codes.htm

ARPA should be valid, but shouldn't be used in any org I think with spire. so easy to check if its been set or not by the end user?